docs/api/covidcast-signals/safegraph.md

```diff
@@ -33,6 +33,10 @@ documentation](https://docs.safegraph.com/docs/social-distancing-metrics).
 | `full_time_work_prop` | The fraction of mobile devices that spent more than 6 hours at a location other than their home during the daytime (SafeGraph's `full_time_work_behavior_devices / device_count`) |
 | `part_time_work_prop` | The fraction of devices that spent between 3 and 6 hours at a location other than their home during the daytime (SafeGraph's `part_time_work_behavior_devices / device_count`) |
 | `median_home_dwell_time` | The median time spent at home for all devices at this location for this time period, in minutes |
+| `completely_home_prop_7dav` | Same as `completely_home_prop` but averaged over a 7 day trailing window. |
```

Comment:
```suggestion
| `completely_home_prop_7dav` | Offers a 7-day trailing window average of the `completely_home_prop`. |
```

Not sure if it could be better worded this way, the suggestion is at the discretion of others. If the rewording changes the meaning or description then please disregard and simply use a hyphen between 7 and day (7-day). If not then perhaps all the others could follow suit.